cmd/podman/buildfarm/buildfarm.go

```diff
@@ -0,0 +1,51 @@
+package buildfarm
+
+import (
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/validate"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Command: podman _buildfarm_
+	buildfarmCmd = &cobra.Command{
+		Use:   "buildfarm",
+		Short: "Farm out builds to remote machines",
+		Long:  "Farm out builds to remote machines that podman can connect to via podman system connection",
+		RunE:  validate.SubCommandExists,
+	}
+)
+
+var (
+	// Temporary struct to hold cli values.
+	buildFarmOpts = struct {
+		Farm  string
+		Local bool
+	}{}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: buildfarmCmd,
+	})
+	buildfarmCmd.Hidden = true
+
+	flags := buildfarmCmd.Flags()
+	podmanConfig := registry.PodmanConfig()
+
+	farmFlagName := "farm"
+	// If remote, don't read the client's containers.conf file
+	defaultFarm := ""
+	if !registry.IsRemote() {
+		defaultFarm = podmanConfig.ContainersConfDefaultsRO.Farms.Default
+	}
+	flags.StringVarP(&buildFarmOpts.Farm, farmFlagName, "f", defaultFarm, "Farm to use for builds")
+
+	localFlagName := "local"
```

Comment:
Comment from the demo meeting:

"local" would confuse me working on a Mac or Windows.  Should we hide the flag for remote clients?